lib/friendly-errors/transformers/missing-loader.js

```diff
@@ -49,6 +49,10 @@ function transform(error) {
         case 'jsx':
             error.loaderName = 'react';
             break;
+        case 'tsx':
+        case 'ts':
+            error.loaderName = 'typescript';
+            break;
```

Comment:
Good catch on this! As it looks like you discovered, this is what gives us the good error when you try to load a `.tsx` file and you haven't enabled its loader :)